src/videojs.ima.js

```diff
@@ -814,6 +834,9 @@
       for (var index in contentEndedListeners) {
         contentEndedListeners[index]();
       }
+      clearInterval(updateTimeIntervalHandle);
```

Comment:
Maybe clear these in the setUpPlayerIntervals_ call to be sure you can't have duplicates?